_posts/en/2019-06-14-exec-briefing.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Bitcoin Optech 2019 Executive Briefing
+permalink: /en/2019-exec-briefing/
+name: 2019-06-14-exec-briefing
+type: posts
+layout: post
+lang: en
+
+excerpt: >
+  Videos and slides from the Bitcoin Optech Executive Briefing
+  on May 15th, 2019.
+
+---
+<style>
+/* For video embeds */
+.container {
+    position: relative;
+    width: 100%;
+    height: 0;
+    padding-bottom: 56.25%;
```

Comment:
56.25% is a very specific number...  :-)

Reply:
This is now removed.

(the specific number was copied and pasted from stack exchange, like all of my html/css :) )